cime_config/testdefs/testmods_dirs/cam/outfrq_hack_shallow_derecho/user_nl_cam

```diff
@@ -29,5 +29,5 @@ hkconv_c0 = 1.0e-4
 ! history fields for CAM4 Hack shallow convection
 hist_output_frequency;h1: 1*nsteps
 hist_max_frames;h1: 1
-hist_add_inst_fields;h1: CMFDT,CMFDQ,CMFDLIQ,CMFDICE,CMFDQR,DQP,ICWMRSH,CMFSL,CMFLQ,FREQSH,EVAPTCM,FZSNTCM,EVSNTCM,HKNTPRPD,HKNTSNPD,HKFLXPRC,HKFLXSNW,HKEIHEAT,EVAPQCM,PRECSH,CMFMCSH,CMFMC,CLDTOP,CLDBOT,PCLDTOP,PCLDBOT,ZMDLF,SHDLF
+hist_add_inst_fields;h1: CMFDT,CMFDQ,CMFDLIQ,CMFDICE,CMFDQR,DQP,ICWMRSH,CMFSL,CMFLQ,FREQSH,EVAPTCM,FZSNTCM,EVSNTCM,HKNTPRPD,HKNTSNPD,HKFLXPRC,HKFLXSNW,HKEIHEAT,EVAPQCM,PRECSH,CMFMCSH
```

Comment:
Since we are modifying this test anyways, can we change the grid to `ne3pg3_ne3pg3_mg37`?  I don't believe you need to modify the snapshot files or anything (the null dycore actually ignores the specified grid), but I have heard discussion about getting rid of the `ne5` grid completely at some point, so this would save us a future hassle.  Thanks!